### PR TITLE
Hello World Project from Module 1

### PR DESCRIPTION
**Pull request by sebalp100**
### **Hello Microverse Project from Module 1**

- Added CSS, HTML and Test file.
- Added Linters to check for errors.
- Edited the README file to better fit the project.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (franclobo) [inline .gitignore]: - [x] Please 🙏🏽 add the `node_modules/` folder to the `.gitignore` file.

File: README.md [hunk 2]
```diff
@@ -1 +1,52 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
Comment (franclobo) [inline README.md]: - [x] Kindly add "Microverse" to the Acknowledgments section. 👍🏽

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @sebalp100, 

Good job so far!
I'm very proud of your work at the first 👏🏽 👏🏽 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct GitHub flow ✔️ 
- Professional README file ✔️ 
- Descriptive PR ✔️ 
- Descriptive commits 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] Add a title to your PR.
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (sebalp100) [reply .gitignore]: Done!

File: README.md (hunk 2 above)
Comment (sebalp100) [reply README.md]: @franclobo  Also done. Do I need to submit a new review request?

**Comment by sebalp100** [comment]
@franclobo Sorry, what do you mean by "Add a title to your PR"? Is the one I used not showing up?

**Comment by franclobo** [comment]
> Is the one I used not showing up?

Thanks for asking @sebalp100 words don't come easy to explain my point, here is a picture 👇🏽 

> Remember to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

image

**Comment by sebalp100** [comment]
> > Is the one I used not showing up?
> 
> image

@franclobo  Oh ok, thank you. Fixed that too. Should I send a new review request now?

**Comment by franclobo** [comment]
> > > Is the one I used not showing up?
> > 
> > 
> > image
> 
> @franclobo Oh ok, thank you. Fixed that too. Should I send a new review request now?

Yes, @sebalp100 great job implementing the required changes 👏🏽 👏🏽 👏🏽. You can submit it now for a re-review. Happy codding! 😃

**Review by mgmediaweb — APPROVED** [review]
Hi @sebalp100, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved5

## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**